<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>42</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Coffee Database Init</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_init">
       <property name="text">
        <string>초기화(&amp;i)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_backup">
       <property name="text">
        <string>백업(&amp;b)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_restore">
       <property name="text">
        <string>복구(&amp;r)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
